building-applications/expression-language.md

```diff
@@ -0,0 +1,73 @@
+# Expression Language
+
+To support [Conditional steps](compute.md#conditional-steps) and the Compute Transform, an expression language is required to evaluate the conditional step `when` or the compute step `expression`. The syntax is [EL](https://javaee.github.io/tutorial/jsf-el001.html#BNAHQ), which uses the dot notation to access field properties or map keys. It supports the following operators and functions:
```

Comment:
This is correct but it doesn't cover all the cases.

I would rephrase it to say that in LangStream we use this language to refer to the parts of the record (link to the page that explain what a record/message is).

For example we use it:
- in the "query" agents (and "query-vector-db")
- in the "vector-db-sink" agents
- in the "dispatch", "flare-controller". "http-request" agents
- in the "compute" agent
- in the "when" clause
.... (probably it is not worth to cite the whole list, as basically almost of the agents use this language)